### ISSUE TITLE
[a11y]: Subrecipients - accessibility findings

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3900]
### Why is this issue important?

This ticket contains multiple accessibility issues in arpa_reporter.  As support for Arpa reporter is winding down we collected issues in one page.  

Note:  These were identified during a developer accessibility review to double check the accessibility of the elements on the page.   To fully insure the site is accessible to users, more comprehensive testing by an accessibility subject matter expert should be performed.

Testing approach
All testing was done against [staging](https://staging.grants.usdr.dev/arpa_reporter) and local dev.
1) Run [axe DevTools chrome extension ](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)against the page.   This checks contrast 
2) Run through the page with the keyboard - I used Microsoft's [Accessibility Insights Chrome Extension](https://chromewebstore.google.com/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni) to track keyboard focus
3) Zoom the page to 200x and 400x to check reflow
4) Run through the page using a screen reader - mac voice over
5) Validate the HTML by pasting parts of the html into [W3C Markup Validation Service](https://validator.w3.org/#validate_by_input)

Testing gaps
1) Windows screen readers - JAWS / NVDA
2) Mobile web devices - the arpa reporter was not designed to be responsive
3) Real user workflow steps.   

----
# Subrecipients View

## issue 1 - color contrast for the Upload id is insufficient

![Image](https://github.com/user-attachments/assets/7d04b5b4-4670-418b-af4c-a0d740764d40)

![Image](https://github.com/user-attachments/assets/640ea99f-7282-4314-94e7-1e4c50d784f3)

recommended fix:  Use the grants blue - see this PR - [a11y(arpa): fix headings and color contrast issues](https://github.com/usdigitalresponse/usdr-gost/pull/3811/files)

## issue 2 - insufficient color contrast for the Archive button when the cell background is grey

![Image](https://github.com/user-attachments/assets/82522cc3-52fa-488f-9832-a5782a0e50e9)

recommended fix:  Use [Contrast Finder](https://app.contrast-finder.org/?lang=en) for suggestions for other colors

## issue 3 - Empty table cell in filter row for Created, Upload ID, and Edit columns - Minor issue

![Image](https://github.com/user-attachments/assets/cccd6737-a423-4185-9743-e8d44fa3032d)

## issue 4 - Edit and Archive buttons should have aria-label that includes an identifier for the row that is being edited.

![Image](https://github.com/user-attachments/assets/e67ec249-212b-4091-bbcf-8d7cbbbbf8ff)

### Current State

When user visits X page...

### Expected State

When user visits X page...

### Implementation Plan

The following functions need to be added...

### Relevant Code Snippets

```shell

```
